includes/models/class-listing.php

```diff
@@ -792,7 +792,10 @@ public function update_plan( $plan = null, $args = array() ) {
 		}
 
 		$row['listing_id'] = $this->id;
-		$row['is_sticky']  = (int) $row['is_sticky'];
+
+		if ( ! empty( $row['is_sticky'] ) ) {
```

Comment:
Should this be isset instead? Just wondering what an empty value or false might do if it gets passed on.

Reply:
Great catch! From that line, I thought it was always either 0 or 1, but I checked in the `class-fee-plan.php` model and you are right, it can also be false so I pushed a commit to change that to isset.

Reply:
This still isn't quite right. The logic looks backwards now with !

Reply:
Right! 🤦‍♂️ 

Apologies for that.

Reply:
Thanks! Looks good now!